docs/application/native/guides/app-management/ui-app/dali-app.md

```diff
@@ -0,0 +1,44 @@
+# DALi Applications
+
+
+You can use the Dynamic Animation Library (DALi) 3D UI toolkit to create a high-performance, rich UI application. For 2D-based Tizen native applications, use the [Enlightenment Foundation Libraries (EFL) UI toolkit](../../ui/efl/introduction-n.md).
```

Comment:
The link is broken. It looks like `../../ui/efl/index.md` is correct.

Reply:
done